recipes/lintquarto/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "lintquarto" %}
+{% set version = "0.4.0" %}
+{% set python_min = "3.7" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.